modules/cnf-provisioning-deploying-a-distributed-unit-(du)-manually.adoc

```diff
@@ -0,0 +1,130 @@
+// CNF-950 4.7 Provisioning and deploying a Distributed Unit (DU) manually
+// Module included in the following assemblies:
+//
+// *cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-provisioning-deploying-a-distributed-unit-manually_{context}"]
+= Provisioning and deploying a Distributed Unit (DU) manually
+
+Radio access network (RAN) is composed of Centralized Units (CU), Distributed Units (DU) and Radio Units (RU).
+RAN from the telecommunications standard perspective is shown below:
+
+image::135_OpenShift_Distributed_Unit.svg[High level RAN overview]
+
+From the three components composing RAN, the CU and DU can be virtualized and implemented as cloud-native functions.
+
+The CU and DU split architecture is driven by real-time computing and networking requirements.
+A DU can be seen as a real-time part of a telecommunication baseband unit.
+One Distributed Unit may aggregate several cells. A CU can be seen as a non-realtime part of a baseband unit, aggregating
+traffic from one or more Distributed Units.
+
+A cell in the context of a DU can be seen as a real-time application performing intensive digital signal processing, data transfer,
+and algorithmic tasks.
+Cells often use hardware acceleration (FPGA, GPU, eASIC) for DSP processing offload, but there are also software-only implementations
+(FlexRAN), based on AVX-512 instructions.
+
+Running cell application on COTS hardware requires the following features to be enabled:
+
+* Real-time kernel
+* CPU isolation
+* NUMA awareness
+* HUGEPAGES memory management
+* Precision timing synchronization using PTP
+* AVX-512 instruction set (for Flexran and / or FPGA implementation)
+* Additional features depending on the RAN operator requirements
+
+Accessing hardware acceleration devices and high throughput network interface cards by virtualized software applications
+requires use of SR-IOV and Passthrough PCI device virtualization.
+
+In addition to the compute and acceleration requirements, DUs operate on multiple internal and external networks.
+
+== The manifest structure
+
+The profile is built from one cluster specific folder and one or more site-specific folders.
+This is done to address a deployment that includes remote worker nodes (several sites belonging to the same cluster).
+
+The [`cluster-config`](ran-profile/cluster-config) directory contains performance and PTP customizations based upon
+operator deployments in [`deploy`](../feature-configs/deploy) folder.
+
+The [`site.1.fqdn`](site.1.fqdn) folder contains site-specific network customizations.
+
+== Prerequisites
+
+Before installing the operators and deploying the DU, perform the following steps.
+
+. Clone the files in the `ran-profile` folder here:
+link:https://github.com/openshift-kni/cnf-features-deploy/tree/master/feature-configs/cn-ran-overlays/ran-profile[ran-profile].
```

Comment:
I dont think we'd want an official documentation pointing to a non-officially released content

Reply:
even if we do, better point to a specific commit and not to master

Reply:
and even that might change.
that why we have an official releasing process..

Reply:
@kenyis Do you have alternative text/link?

Reply:
https://github.com/openshift-kni/cnf-features-deploy/tree/release-4.7/feature-configs/cn-ran-overlays/ran-profile

This is the specific branch for 4.7.  This will not change over time.

Reply:
Commit:
https://github.com/openshift-kni/cnf-features-deploy/commit/60d341206e4a4dd88bc14c903c3b42cdf5adbdc3

Reply:
Let's remove the text about cloning the repo

Reply:
@vitus133 Removed this step: Clone the files in the `ran-profile` folder:
link:https://github.com/openshift-kni/cnf-features-deploy/tree/master/feature-configs/cn-ran-overlays/ran-profile[ran-profile].